crates/typst/src/eval/methods.rs

```diff
@@ -172,7 +172,15 @@ pub fn call(
             "keys" => dict.keys().into_value(),
             "values" => dict.values().into_value(),
             "pairs" => dict.pairs().into_value(),
-            _ => return missing(),
+            _ => {
+                return if dict.contains(method)
+                    && matches!(dict.at(method, None).unwrap(), Value::Func(_))
```

Comment:
Perhaps this could be simplified to the following?
```suggestion
                return if matches!(dict.at(method, None).unwrap(), Ok(Value::Func(_)))
```